test/support/annotations/annotation_writer.rb

```diff
@@ -0,0 +1,36 @@
+module Annotations
+  class AnnotationWriter
+    def header_template
+      File.read(File.dirname(__FILE__) + '/templates/header.erb')
+    end
+
+    def annotation_template
+      File.read(File.dirname(__FILE__) + '/templates/annotation.erb')
+    end
+
+    def initialize(filename)
+      @file = File.open(filename, 'w')
+      @file.write(header_template)
+    end
+
+    def close
+      @file.close
+    end
```

Comment:
Feels dangerous to have to explicitly remember to call to close the file. I am guessing you are doing this to be able to append to the file? And you want to do that in memory rather than writing a function that opens, writes in append mode and closes?